### PR TITLE
Set webpack to have custom props for node and browser

### LINKED ISSUE #18: Webpack should compile server and client-side files differently
When using node-fetch or any other package that have custom behavior for differents JS environment the compilation of `core` and `ui` files should be handled differently, each one using their own particularities.

### PR DESCRIPTION
**Pull request by rafaelverger**
Closes #18

### CODE REVIEW

File: webpack.config.js [hunk 1]
```diff
@@ -34,8 +31,35 @@ module.exports = {
 
   plugins: [
     new CheckerPlugin(),
+    /*
```
Comment (andersonba) [inline webpack.config.js]: forgot to remove these?

File: webpack.config.js [hunk 2]
```diff
@@ -1,24 +1,21 @@
...
+const webpack = require('webpack');
+
 const { CheckerPlugin } = require('awesome-typescript-loader')
 
 const srcPath = path.join(__dirname, '/src');
 const libPath = path.join(__dirname, '/lib');
 
-module.exports = {
-  context: srcPath,
+const nodeExternals = require('webpack-node-externals');
```
Comment (andersonba) [inline webpack.config.js]: missing in package.json/yarn.lock

Comment (rafaelverger) [reply webpack.config.js]: fixed

File: webpack.config.js (hunk 1 above)
Comment (rafaelverger) [reply webpack.config.js]: yeah :)

**Comment by rafaelverger** [comment]
@andersonba apparently only lodash is needed as a "real" dependency

**Pull request merged**